### ISSUE TITLE
Additional validations needed on the donation page 

### ISSUE THREAD

**Issue by aarjay** [HTBox/crisischeckin#536]
Bug entered by Valerie Campbell - valerie.campbell@atlasair.com
Tested in Chrome OS Version 49.0.2623.112 (64-bit) on Windows 

Issue:
1. Page accepts single letter for first name and last name, even after 'Review Donation and Continue' is clicked.
2. Donation amount, credit card, expiration date, phone number and CSC fields validate only after 'Review Donation and Continue' is clicked. These fields should not allow characters other than numbers.
3. If invalid phone number is entered, the Phone Number field shows red but the field is not listed in the error message text. 
4. The Phone Number field does not accept international phone number format. Is that by design? 
Attached screenshot covers 1-4 above.
![screenshot 2016-05-04 at 5 44 30 pm](https://cloud.githubusercontent.com/assets/5223163/15030378/e6412a9e-121f-11e6-8a90-eec033c82df2.png)

5. City, State and Zip fields are listed in the error message as invalid when any one field is incorrect. (screenshot attached) Is that by design?
![screenshot 2016-05-04 at 5 36 05 pm](https://cloud.githubusercontent.com/assets/5223163/15030267/51467f16-121f-11e6-9391-a08846fa825c.png)

**Comment by NaveenP94** [comment]
What donation page is this?

**Comment by tonysurma** [comment]
Thanks for the issue, this looks like bugs on the paypal site so out of scope for us
